### ISSUE TITLE
Support for LCVR mod (if possible?)

### ISSUE THREAD

**Issue by StupidRepo** [Wet-Boys/LethalEmotesAPI#183]
Afaik, the only button that isn't used by the mod is the X button (on a Meta Quest 3 controller, idk what that button would be for other headset controllers) so the player could hold down that button to open the emote wheel then move their controller in the direction of the emote they want on the wheel.

Implementing opening the menu for the VR mod should be easy, and LCVR provides UI handling by default but I don't know if that'll work for mods too. Either way, I would love to see this mod support LCVR, unless it does already (in which case please delete this issue 😭)

Edit: Actually I think this mod just normally supports LCVR (as in, not supported by devs but works anyway)... also I haven't tested this yet as I only thought about this and now cba to put my headset back on.

**Comment by ToastedOven** [comment]
Yeah last I checked, emotesapi does "work" with lcvr, however we reached out to them before about implementing inpututils so we can actually bind stuff and they ghosted us on it. Not much to do there

**Comment by ToastedOven** [comment]
If it doesn't work in any capacity let me know ( the random key should at least work) and I'll fix it, but I can't do anything about keybinds (I mean technically we could, but it's not worth it if they won't meet us halfway)

**Comment by StupidRepo** [comment]
Alright thanks. Will probably test in a min or later tonight. I know in their compat docs they mentioned OpenVRToKey for other mods. Might try that if I'm having problems but idk if that'll work.

Thing is, there's just not enough buttons for the base game binds and then extra mod binds. Anyway, I'll lyk the results. 😂

**Comment by StupidRepo** [comment]
@ToastedOven Yep, random emotes button works. Emotes wheel doesn't show. Animated character doesn't appear on left hand either :(
